src/pages/mentor-registration/index.js

```diff
@@ -0,0 +1,37 @@
+import { useTranslation } from 'react-i18next'
+import PrimaryButton from '../../components/buttons/primary'
+import { InputSearch } from '../../components/inputs/search'
+import { Table } from '../../components/table/table'
+import { Responsive, Container } from './styled.js'
+
+export const MentorRegistrationPage = () => {
+  const { t } = useTranslation()
+  const handleSubmit = () => { }
+
+  return (
+    <Responsive>
+    <div className="page-container">
+      <section>
+        <h1>{t('user.title')}</h1>
+        <InputSearch/>
+        <div className ='button'>
+          <PrimaryButton text={t('user.newMentor.text')} onClick={handleSubmit} />
```

Comment:
O Link q usamos é a tag <Link to=“” />

Reply:
E ele nao pode ser fixado para acelera-mais, ele precisa ser relativo ao site do ambiente